### ISSUE TITLE
[Suggestion] Alveary Gearbox

### ISSUE THREAD

**Issue by jjc54** [ForestryMC/ForestryMC#1773]
Gearboxes currently act as a source of power for farm and greenhouse multiblocks. However, there is no gearbox for alvearys and a power source must be connected to every single alveary block if you wish it to be powered. The piping/generators can get quite crazy when you have a ton of alveary upgrades in order to breed bees from a different biome. Adding a gearbox for alvearys would fix this problem, and it feels like something that Forestry should of always had.

**Comment by mezz** [comment]
This exists in Binnie's Mods already, actually: https://binnie.mods.wiki/wiki/Alveary_Transmission

**Comment by jjc54** [comment]
Oh yeah I wasn't taking Binnies into consideration since this just felt like something vanilla forestry would have.

**Comment by NightKosh** [comment]
I'm agreed with @whitestarx0312 that such feature should be a part of "vanilla Forestry". It is increadibly annoyng to use another addon mod just for 1 - 2 items, to make vanilla items works as they should by default. At the same time all those addons provide a lot of additional items which doesn't required at all. 
But the most terrible part of such practice occures when one of such addons incompatible with other mod, or even doesn't updated for a long time, which happens rather frequently.

**Comment by SandGrainOne** [comment]
Mods should not add stuff "in case another mod might need it". The question should be: Does Forestry need a gearbox in and of itself. 

There are 2 Alveary upgrades in Forestry that use power. The heater and the fan. There will be cases where you need more than one heater or fan so the idea of a gearbox is not bad. Though personally I don't consider it too much of an issue that I sometimes must use 2-3 extra cables/pipes to power more than one machine...

**Comment by NightKosh** [comment]
@SandGrainOne I agree that mods should not add stuff just because another mod might need it. But this feature required for "vanilla" stuff too, as I think. 
As you already said that there are 2 upgrades in Forestry which may need it, and it is rather common case to use both of them in one alveary. As for me - the third alveary I build every time has 2 "Heaters" 2 "Fans" and 2 "Hygroregulators", to make it easier to breed some bees with specific gens while it can not be breed in my main base biome.

**Comment by SandGrainOne** [comment]
I didn't think the Alveary supported having both active Fans and active Heaters. I would assume they nullified each other. Hygroregulator do not require power. But this is nitpicking. I achnowledge that a gearbox could be convenient in a few cases.

**Comment by NightKosh** [comment]
@SandGrainOne for sure I didn't use both of them at the same time - any of them can be disabled by lever. I'm just describing one of the most problematic situation(which happens for me) - I prefer to have 1 - 2 "technical" alvearies which built of all those components, for "specific" bee breeding, instead of replace blocks in other alvearies all the time.
